doc/user-docs/index.md

```diff
@@ -87,4 +87,12 @@ For an introduction to the various settings, see [Using Json Settings](UsingJson
 2. Terminal zoom can be changed by holding <kbd>Ctrl</kbd> and scrolling with mouse.
 3. Background opacity can be changed by holding <kbd>Ctrl</kbd>+<kbd>Shift</kbd> and scrolling with mouse. Note that acrylic transparency is limited by the OS only to focused windows.
 4. Open Windows Terminal in current directory by typing `wt -d .` in the address bar.
-5. Please add more Tips and Tricks.
+5. Random theme in terminal by using Zsh(Z-shell) for Ubuntu distro [LearnMoreAboutZ-shell](https://github.com/ohmyzsh/ohmyzsh)
+* `sudo apt install zsh` 
+* `sh -c "$(curl -fsSL https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"`
+* `sudo apt-get install fonts-powerline`
+* `code ~/.zshrc` # to customize it
+* Then change the `ZSH_THEME="random"`. Your terminal sessions always having unique theme.
+6. Please add more tips & tricks
```

Comment:
Forgot the period at the end.

Reply:
Sorry but What do you mean by period I didn't get it ?

Reply:
> Please add more tips & tricks

Please add more tips & tricks.

Reply:
I think you didn't seen last six number point I already mentioned there.about please add more tips and tricks. Thank you 😊